packages/apify/src/crawlers/cheerio_crawler.ts

```diff
@@ -926,3 +919,10 @@ export class CheerioCrawler extends BasicCrawler {
         }
     }
 }
+
+interface RequestFunctionOptions {
```

Comment:
Should probably export this? CC @B4nan thoughts?

Reply:
idk, given its used only in protected method probably not?

btw please use `;` instead of `,` inside types. i just added eslint rule to master, it has a fixer, so you can just rebase the changes from master and run `lint --fix`

Reply:
kept without export. linting - done.